src/NightwatchServiceProvider.php

```diff
@@ -184,6 +184,7 @@ private function registerLogger(): void
             $this->config->set('logging.channels.nightwatch', [
                 'driver' => 'custom',
                 'via' => Logger::class,
+                'level' => $this->config->get('nightwatch.filtering.log_level'),
```

Comment:
We should use the already resolved Nightwatch config, which will have a small performance gain.

```suggestion
                'level' => $this->nightwatchConfig['filtering']['log_level'] ?? 'debug',
```

Reply:
We should also update the type of the `private array $nightwatchConfig` property to include this new value.